bin/fin

```diff
@@ -5567,7 +5569,12 @@ ngrok_share ()
 
 	# Add -host-header by default if no arguments are included.
 	if [[ "${ARGS}" == "ngrok http" ]]; then
-		ARGS="${ARGS} -host-header '${NGROK_VIRTUAL_HOST}' ${container_name}.${network}:${NGROK_PORT}"
+		ARGS="${ARGS} -host-header '${NGROK_VIRTUAL_HOST}'"
+	fi
+
+	# If no configuration was used we append main container.
```

Comment:
Could you please explain better why we only append main container if CONF_ARGS is empty.
I am asking because I genuinely have no idea why myself :)

Reply:
When you use configuration files you can't also specify the container. It's an either or situation. So, if there is no configuration files we append the container on the end.

Reply:
Why? Because its ngrok limitation, or because of logical conflict?

Reply:
@achekulaev this is an ngrok limitation. So far I have found that it uses either the config files or it uses the command line arguments.